assets/pages/contributions-landing/newContributionsLanding.jsx

```diff
@@ -82,6 +82,122 @@ const content = (
     <Page header={<header />} footer={<footer />}>
       <h1>{countryGroupSpecificDetails[countryGroupId].headerCopy}</h1>
       <p className="blurb">{countryGroupSpecificDetails[countryGroupId].contributeCopy}</p>
+      <form action="#" method="post" className="form form--contribution">
```

Comment:
It's a lot of JSX inside a single component. And plenty of room for abstraction in terms of repetitive elements (e.g. amounts fields). I'm guessing you're planning to split out into subcomponents later?

Reply:
> I will add logic in follow-up PRs, just want to get the rendering done so it's out of the way.

Just re-read that - pretty much answers my question